code/build.gradle

```
@@ -115,20 +115,20 @@ java {
 publishing {
     publications {
         maven(MavenPublication) {
-            groupId = "io.github.pm-dungeon"
-            artifactId = "core"
+            groupId = "io.github.programmiermethoden"
+            artifactId = "pm-dungeon"
             version = project.version
 
             from components.java
 
             pom {
                 name = "PM-Dungeon core"
```

was ist damit ("core")?

---

Moin, das ist eigentlich nur eine Bezeichnung, aber das " core" sollte weg.

```suggestion
                name = "PM-Dungeon"
```